interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -1649,6 +1649,12 @@ namespace cling {
     // Return a symbol's address, and whether it was jitted.
     std::string mangledName;
     utils::Analyze::maybeMangleDeclName(GD, mangledName);
+#if defined(LLVM_ON_WIN32)
+    if (mangledName.size() > 2 && mangledName[1] == '?' &&
+        mangledName.find(std::string("?__cling_Un1Qu")) == std::string::npos) {
```

Comment:
`find` needs to look at the whole string. Does the string always start with `"?__cling_Un1Qu"`? Can be use `compare(..., 15)` or whatever the correct string length is? :-)

Reply:
Do we know why `"?__cling_Un1Qu"` is special? I bet it isn't - it's just the example of a certain case that we see, and instead of looking for the string we should be looking for the case. `extern "C"` maybe?

Reply:
For the record: There is a special symbol `'\01'` added in front of the mangled names by clang 5.0 on Windows. AFAIK this is not the case anymore with clang 8.0. And OK, I'll use `compare()` and I'll squash the commits once I have your green light